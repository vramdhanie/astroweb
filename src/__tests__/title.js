import React from 'react'
import renderer from 'react-test-renderer'
import Title from '../components/title'

describe('Title', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Title title="Hello" subtitle="World" />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})