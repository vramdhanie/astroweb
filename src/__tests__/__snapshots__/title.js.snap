// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<div
  className="sc-bdVaJa djIATI"
>
  <h4>
    <span
      className="title"
    >
      Hello
    </span>
    <span>
      World
    </span>
  </h4>
</div>
`;
